### PR TITLE
data:JenilGajjar20

### PR DESCRIPTION
**Pull request by JenilGajjar20**

### CODE REVIEW

File: data/JenilGajjar20.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/JenilGajjar20",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Follow me on Medium",
+      "url": "https://jenilgajjar.medium.com/",
+      "icon": "FaMedium"
+    },
```
Comment (Aadarsh805) [inline data/JenilGajjar20.json:27]: hey this comma at the end of line 27 needs to be removed

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

File: data/JenilGajjar20.json (hunk 1 above)
Comment (JenilGajjar20) [reply data/JenilGajjar20.json]: Okay, change Updated

**Review by Aadarsh805 — APPROVED** [review]
All good now

**Review by dan-mba — APPROVED** [review]
Looks good to me

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good to me! 👍 
Great collaboration Everyone! 🎉

**Pull request merged**